### PR TITLE
Resizeinputs refactoring

### PR DESCRIPTION
**Pull request by YaroslavNikiforov**

### CODE REVIEW

**Comment by YaroslavNikiforov** [comment]
if you call resizeInputs in componentDidUpdate it stars to lag

File: front-end/src/helpers/resizeInput.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
-const resizeInput = (el) => {
+const resizeInput = (arr) => {
```
Comment (Thealoner) [inline front-end/src/helpers/resizeInput.js]: I'm thinking resizeInputs (with 's') would be a better name, that would suggest it accepts an array.

**Comment by Thealoner** [comment]
> if you call resizeInputs in componentDidUpdate it stars to lag

Do you know why? Does it trigger another render?

**Comment by YaroslavNikiforov** [comment]
I think that something is wrong with out RecordEditor component. Because, regardless of the use of this function, when it renders at first - render function is calling five times and componentDidUpdate four times

**Pull request merged**